CHANGELOG.rst

```diff
@@ -1,5 +1,12 @@
 Changelog for Mantarray Frontend Components
 ===========================================
+
+0.3.0 (unreleased)
+------------------
+
+- Added heatmap
+- Added websocket to receive waveform data asynchronously and in real time
+
```

Comment:
Not sure what else needs to be put in this, I think @jperson mentioned there were some new y-axis zoom features that have been implemented

Reply:
@tannermpeterson yeah, it can say `Added y-axis-zoom for waveform display`

Reply:
I also need to add to this..